ksp_plugin/manœuvre_body.hpp

```diff
@@ -43,6 +43,12 @@ Instant Manœuvre<InertialFrame, Frame>::final_time() const {
 }
 
 template<typename InertialFrame, typename Frame>
+bool Manœuvre<InertialFrame, Frame>::FitsBetween(Instant const& begin,
+                                                 Instant const& end) {
+  return begin < initial_time() && final_time() < end;
```

Comment:
These aren't iterators, they're (floating-point) quantities; I see no reason to follow conventions in use for discrete ranges in STL. As for which one we want, we probably don't care, open sets are nice.